Backpack/src/main/res/drawable/bpk_bg_dialog_bottom.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looking at other resources that are not "external" like `chip_selected` we are not naming them `bpk_` and we are also following `<component_name>_<name>` e.g. `dialog_bg_bottom`. Can we change this to make it consistent?

Reply:
We can, but I'd prefer to keep `bpk_` prefix and it defines the scope of a resource and prevents accidental usage.

Reply:
Can we at least name it `bpk_dialog_bg` to prefix with the component name first

Reply:
Done